pkg/services/move.go

```diff
@@ -36,3 +38,9 @@ type MoveRouter interface {
 	Submit(move *models.Move) error
 	SetLogger(logger Logger)
 }
+
+// MoveWeights is the exported interface for flagging a move with an excess weight risk
+//go:generate mockery --name MoveWeights --disable-version-string
+type MoveWeights interface {
```

Comment:
Someone please suggest a better name 😆

Reply:
```suggestion
type MoveVerify interface {
```

Reply:
My suggestion to `Verify` is that we may use this interface for other types of verifications in the future. I'm using `Check` as the synonym here. Some more examples include: **make sure**, confirm, corroborate, validate, substantiate.

Reply:
For the excess weight field I used the term `qualify`.  For more context I could see this containing the code to auto trigger a reweigh request when this excess weight threshold is met.  To me those are both things that deal with the shipment weights of an entire move.

Reply:
@duncan-truss interesting... So I went with `MoveVerify` because of the name of the service but maybe we should have something like:

```suggestion
type MoveQualifyWeights interface {
```

Reply:
I'm going to leave it alone for now and let the method names do the hardwork.  It should be easy to change as we figure out where things will live and do.  Thanks for the review @rogeruiz !